app/src/renderer/js/main.js

```diff
@@ -331,6 +332,10 @@ document.addEventListener('DOMContentLoaded', () => {
     this.classList.toggle('is-active');
   };
 
+  progressBarCancelBtn.onclick = function () {
```

Comment:
Arrow-function, since `this` is not used.

Reply:
Actually, let's start using `.addEventListener()`. It's better.

Reply:
Will do 👌